### PR TITLE
Greeting Screen Design Complete

### LINKED ISSUE #4: Greeting Screen Design
* Design and create greeting text

    Image

* Design and create the button to navigate to the login/register screen

    

  Image`

* Greetings screen preview

   Image

### PR DESCRIPTION
**Pull request by Manz1999**
# Description
### What?
- Designed and created Greeting Screen
- Designed and recreated the design of the screen accorded to the proposed style. 
- See #4  for more information
### Why
- Design the screens accorded to the proposed design and ensure the functionalities of the application. See #4 for more information.

## Testing

- Since the project is just initialized you only need install the basic packages to visualize the application
```npm install```
- then you can initialize with an android emulator or via web
```npm start```
- or if you use expo
```npx expo start```

## Screenshots

| Proposed design   |      Successfull design      |
|----------|:-------------:|
| proposed | succesfull|

### CODE REVIEW

File: src/styles/GreetingScreen/Screen_Greeting.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    fontWeight: 'bold'
+  },
+
+  text2: {
+    color: 'white', 
+    fontFamily: 'Nunito_500Medium',
+    fontSize: 30, 
+    fontWeight: 'bold',
+    marginLeft:35
+  },
```
Comment (yovanha) [inline src/styles/GreetingScreen/Screen_Greeting.jsx]: You have here duplicated code, lets use the text1 styles in your tex2 and just add the marginLeft style to `text2`

File: src/styles/GreetingScreen/Screen_Greeting.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    alignSelf: 'center',
+    marginTop: 90
+  },
+
+  containerTexts: {
+    marginTop: 130
+  },
+
+  text1: {
+    color: 'white',
```
Comment (yovanha) [inline src/styles/GreetingScreen/Screen_Greeting.jsx]: You have this color in your library colors, could you please add it here?

File: src/styles/GreetingScreen/Screen_Greeting.jsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    backgroundColor: 'white',
+    borderRadius: 30,
+    marginTop: 180,
+    padding: 20,
+    width:'55%' ,
+  },
+
+  textButton: {
+    alignSelf:'center',
+    color: 'darkslateblue',
```
Comment (yovanha) [inline src/styles/GreetingScreen/Screen_Greeting.jsx]: Do you have this color in you library colors?

File: src/styles/LoginScreen/Components_Login.jsx [hunk 4]
```diff
@@ -10,33 +9,44 @@ export const containers = StyleSheet.create({
...
+    backgroundColor: COLORS.WHITE,
+    borderWidth: 1,
+    width: 330,
+    height: 45,
+    padding: 10,
+    fontSize: 13,
+    fontWeight: '300',
+    marginBottom: 10,
+    borderRadius: 8,
     top: 15,
```
Comment (yovanha) [inline src/styles/LoginScreen/Components_Login.jsx]: Is there a reason why did you delete the code which you already refactored?

File: src/styles/LoginScreen/Components_Register.jsx [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+    borderWidth: 1,
+    width: 330,
+    height: 45,
+    padding: 10,
+    fontSize: 13,
+    fontWeight: '300',
+    marginBottom: 10,
+    borderRadius: 8,
+    top: 15,
+  },
```
Comment (yovanha) [inline src/styles/LoginScreen/Components_Register.jsx]: Could you please refactor this? I see you have the same props but you're just changing the bottom and top styles

**Review by yovanha — DISMISSED** [review]

**Review by yovanha — CHANGES_REQUESTED** [review]
Requested changes

**Review by InGerardo777 — APPROVED** [review]
LGTM🍆

**Review by arthurchina92 — APPROVED** [review]
🚀

**Review by IvanGodinez21 — APPROVED** [review]

**Review by FloresEsme — APPROVED** [review]

**Review by yovanha — APPROVED** [review]

**Pull request merged**